pantab/newsfragments/40.bug

```diff
@@ -0,0 +1 @@
+Added "package_data={"": ["*.h"]}" to main setup call so that header files are correctly copied during build.
```

Comment:
```suggestion
Fixed issue where source installations would error with `fatal error: tableauhyperapi.h: No such file or directory`
```

whatsnew notes should avoid being technical; want to think on terms of what a end user may have experienced